screen/ui-auth/src/main/java/kasem/sm/ui_auth/login/LoginScreen.kt

```diff
@@ -7,29 +7,38 @@ package kasem.sm.ui_auth.login
 import androidx.compose.material.SnackbarHostState
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
+import com.ramcosta.composedestinations.annotation.Destination
+import com.ramcosta.composedestinations.spec.DestinationStyle
+import kasem.sm.ui_core.CommonNavigator
+import kasem.sm.ui_core.NavigationEvent
 import kasem.sm.ui_core.rememberStateWithLifecycle
 import kasem.sm.ui_core.safeCollector
 
+@Destination(style = DestinationStyle.BottomSheet::class)
 @Composable
 fun LoginScreen(
     viewModel: LoginVM,
-    onLoginSuccess: () -> Unit,
```

Comment:
Great. A Lot of composable have been cleaned up. Thank You!

Reply:
Glad you liked it